data/custom_content/with_config/bargraphwcustomcolors/multiqc_config.yaml

```diff
@@ -0,0 +1,27 @@
+custom_data:
+    testA:
+        parent_id: parentid
+        parent_name: "tests"
```

Comment:
These aren't doing anything in this example, so I think it's probably clearer to strip them out..

```suggestion
```